tomcat/src/main/java/nextstep/jwp/model/User.java

```diff
@@ -1,7 +1,7 @@
 package nextstep.jwp.model;
 
-import java.util.Map;
-import org.apache.util.UrlUtil;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.util.StringUtil;
```

Comment:
import문을 아래와 같이 축약할 수 있겠네요!
```suggestion
import org.apache.util.StringUtil;
```

Reply:
👍👍👍